src/GitVersion.Core/Configuration/ConfigurationFileLocator.cs

```diff
@@ -55,6 +55,21 @@ public GitVersionConfiguration ReadConfig(string workingDirectory)
         return new GitVersionConfiguration();
     }
 
+    public IDictionary<object, object?> ReadConfiguration(string workingDirectory)
```

Comment:
Good point. I think `ReadConfig` should be removed because it doesn't handle this scenario https://github.com/GitTools/GitVersion/pull/3226#discussion_r994267957. At this moment it used only in the init routine which should be refactored here https://github.com/GitTools/GitVersion/issues/3229.

Reply:
I see.